ui/src/external/monaco.flux.ts

```diff
@@ -0,0 +1,39 @@
+import initialize from 'src/external/monaco'
+
+initialize().then((monaco: any) => {
+  monaco.languages.register({id: 'Flux'})
+
+  monaco.languages.setMonarchTokensProvider('Flux', {
```

Comment:
is this what we connect to wasm later?

Reply:
not exactly. these are all for static syntax things. We pass props to Editor for things that we'll get from wasm